contracts/lbp/LBPManager.sol

```diff
@@ -0,0 +1,290 @@
+/*
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// solium-disable linebreak-style
+pragma solidity 0.8.6;
+
+import "openzeppelin-contracts-sol8/token/ERC20/IERC20.sol";
+import "../utils/interface/ILBPFactory.sol";
+import "../utils/interface/IVault.sol";
+import "../utils/interface/ILBP.sol";
+
+/**
+ * @title PrimeDAO LBPManager contract
+ * @dev   Smart contract for managing a Balancer LBP.
```

Comment:
How would this be?
```
/**
 * @title LBPManager contract
 * @dev   Smart contract for managing interactions with a Balancer LBP.
 */
```

I removed the PrimeDao reference because of a comment from Doug in last weeks call where he said it would be nice to remove the reference in all the contract to make it more generic